curvefs/src/metaserver/s3/metaserver_s3_adaptor.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ *  Copyright (c) 2020 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 2021-8-13
+ * Author: chengyi
+ */
+
+#include "curvefs/src/metaserver/s3/metaserver_s3_adaptor.h"
+
+namespace curvefs {
+namespace metaserver {
+void S3ClientAdaptorImpl::Init(const S3ClientAdaptorOption& option,
+                               S3Client* client) {
+    blockSize_ = option.blockSize;
+    chunkSize_ = option.chunkSize;
+    client_ = client;
+}
+
+int S3ClientAdaptorImpl::Delete(const Inode* inode) {
```

Comment:
define this function proto like this "int S3ClientAdaptorImpl::Delete(const Inode &inode)",
1.  This function do not need to update any members of Inode.
2.  no need to copy s3chunkinfolist here,  if object in s3chunkinfolist  is not exist on s3, also should return OK.

Reply:
fixed